worker/uniter/runner/jujuc/secret-info-get.go

```diff
@@ -82,6 +82,26 @@ type metadataDisplay struct {
 	RotatePolicy     secrets.RotatePolicy `yaml:"rotation,omitempty" json:"rotation,omitempty"`
 	LatestExpireTime *time.Time           `yaml:"expiry,omitempty" json:"expiry,omitempty"`
 	NextRotateTime   *time.Time           `yaml:"rotates,omitempty" json:"rotates,omitempty"`
+	Grants           []GrantInfo          `yaml:"grants,omitempty" json:"grants,omitempty"`
```

Comment:
"access" here too. I think even the struct attribute should be `Access` not `Grants`